Wiley Online Library.js

```diff
@@ -229,7 +229,14 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				'&fileFormat=REFERENCE_MANAGER' +
 				'&hasAbstract=CITATION_AND_ABSTRACT';
 	ZU.doPost(postUrl, body, function(text) {
-		//Z.debug(text)
+		//on at least one occasion, Wiley puts RIS within RIS, thus causing havoc.
+		//This is trying to prevent this (see last test).
```

Comment:
The "last test" might not be always the same... Maybe, point here directly to the url?